src/lib/roller.ts

```diff
@@ -1,6 +1,18 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
This eslint-disable line can be removed - we just needed it while roller wasn't implemented